docs/resources/hyperdrive_config.md

```diff
@@ -0,0 +1,90 @@
+---
+page_title: "cloudflare_hyperdrive_config Resource - Cloudflare"
+subcategory: ""
+description: |-
+  The Hyperdrive Config https://developers.cloudflare.com/hyperdrive/ resource allows you to manage Cloudflare Hyperdrive Configs.
+---
+
+# cloudflare_hyperdrive_config (Resource)
+
+The [Hyperdrive Config](https://developers.cloudflare.com/hyperdrive/) resource allows you to manage Cloudflare Hyperdrive Configs.
+
+## Example Usage
+
+```terraform
+# Config using all default values
+resource "cloudflare_hyperdrive_config" "no_defaults" {
+  account_id = "f037e56e89293a057740de681ac9abbe"
+  name       = "my-hyperdrive-config"
+  password   = "my-password"
+  origin     = {
+    database = "postgres"
+    host     = "my-database.example.com"
+  }
+}
+
+# Config not using any default values
+resource "cloudflare_hyperdrive_config" "no_defaults" {
+  account_id = "f037e56e89293a057740de681ac9abbe"
+  name       = "my-hyperdrive-config"
+  password   = "my-password"
+  origin     = {
+    database = "postgres"
+    host     = "my-database.example.com"
+    port     = 5432
+    scheme   = "postgres"
+    user     = "my-user"
+  }
+  caching    = {
+    disabled               = false
+    max_age                = 60
+    stale_while_revalidate = 30
+  }
+}
+```
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `account_id` (String) The account identifier to target for the resource.
+- `name` (String) The name of the Hyperdrive configuration.
+- `origin` (Attributes) The origin details for the Hyperdrive configuration. (see [below for nested schema](#nestedatt--origin))
+
+### Optional
+
+- `caching` (Attributes) The caching details for the Hyperdrive configuration. (see [below for nested schema](#nestedatt--caching))
+- `id` (String) The identifier of this resource. This is the hyperdrive config value.
+
+<a id="nestedatt--origin"></a>
+### Nested Schema for `origin`
+
+Required:
+
+- `database` (String) The name of your origin database.
+- `host` (String) The host (hostname or IP) of your origin database.
+- `password` (String, Sensitive) The password of the Hyperdrive configuration.
+
+Optional:
+
+- `port` (Number) The port (default: 5432 for Postgres) of your origin database. If not specified, defaults to `5432`.
+- `scheme` (String) Specifies the URL scheme used to connect to your origin database. If not specified, defaults to `postgres`.
+- `user` (String) The user of your origin database. If not specified, defaults to `postgres`.
```

Comment:
```suggestion
- `user` (String) The user of your origin database.
```
This and the other values should be in `Required`. Working on getting the public API spec updated.

Reply:
Done now @OilyLime

Reply:
Public API spec updated.